lib/Ohmbrewer_Screen.cpp

```diff
@@ -31,7 +33,11 @@ Ohmbrewer::Screen::Screen(uint8_t CS,
 
     // Build the menu tree
     _homeMenu = new MenuHome(this, settings);
-    _homeMenu->addChild(new MenuWiFi(this, settings));
+    _mainMenu = new MenuMain(this, settings);
```

Comment:
In the next batch of menus you might want to refactor out this tree assignment into its own function that gets called from the constructor.